pkg/ansible/flags/flag.go

```diff
@@ -0,0 +1,37 @@
+package flags
+
+import (
+	"strings"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/ansible/controller"
+	"github.com/spf13/pflag"
+)
+
+// AnsibleOperatorFlags - Options to be used by an ansible operator
+type AnsibleOperatorFlags struct {
+	ReconcilePeriod time.Duration
+	WatchesFile     string
+}
+
+// ApplyTo - Apply the flags to the controller options.
+func (f *AnsibleOperatorFlags) ApplyTo(options *controller.Options) {
```

Comment:
This is definitely a nit but `ApplyTo` seems too generic of a name for this. Are there any other controller options that we intend to expose as flags? If so then I can see why we would want to keep it generic, but it feels weird to me that we are in the `flags` package and the function is just called `ApplyTo`. I would prefer if it was named `ApplyToController` or something that specified that this is applying flag options onto the controller.

Reply:
I only mention it because line 48 in `operator.go` didn't make sense to me at first.